src/main/webapp/WEB-INF/deployerConfigContext.xml

```diff
@@ -54,12 +51,106 @@
           c:authenticator-ref="authenticator">
         <property name="principalAttributeMap">
             <map>
+            </map>
+        </property>
+    </bean>
+
+    <bean id="attributeRepository"
+          class="org.jasig.services.persondir.support.ldap.LdaptivePersonAttributeDao"
+          p:connectionFactory-ref="ldap.connection.factory.pooled"
+          p:baseDN="ou=sso,ou=account,dc=ccci,dc=org"
+          p:searchControls-ref="searchControls"
+          p:searchFilter="cn={0}">
+        <property name="resultAttributeMapping">
+            <map>
+                <!--
+                   | Key is LDAP attribute name, value is principal attribute name.
+                   -->
                 <entry key="relayGuid" value="id" />
```

Comment:
we should match our attribute release names to what they are in our actual CAS servers to make the transition to the actual CAS server easier down the line.